hw/ip/spi_device/doc/checklist.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "SPI_DEVICE Checklist"
+---
+
+This checklist is for [Hardware Stage]({{< relref "/doc/project/hw_stages.md" >}}) transitions for the [spi_device peripheral.]({{< relref "index.md" >}})
+All checklist items refer to the content in the [Checklist]({{< relref "/doc/project/checklist.md" >}}).
+
+## Design Checklist
+
+### D1
+
+Type          | Item                  | Resolution  | Note/Collaterals
+--------------|-----------------------|-------------|------------------
+Documentation | [SPEC_COMPLETE][]     | Done        | [SPI_DEVICE Spec][]
+Documentation | [CSR_DEFINED][]       | Done        |
+RTL           | [CLKRST_CONNECTED][]  | Done        |
+RTL           | [IP_TOP][]            | Done        |
+RTL           | [IP_INSTANCED][]      | Done        |
+RTL           | [MEM_INSTANCED_80][]  | Done        | `prim_ram_2p` is instantiated
+RTL           | [FUNC_IMPLEMENTED][]  | Done        | Firmware Operation Mode only
+RTL           | [ASSERT_KNOWN_ADDED][]| Done        |
```

Comment:
Is this one done? Seems like interrupt output and spi_dev output haven't been checked

Reply:
Uh... I thought I added. Thanks I will create another PR